### ISSUE TITLE
RGB cursor position stuck

### ISSUE THREAD

**Issue by ddejohn** [kprinssu/anne-keyboard-windows#38]
Cursor remains fixed in the top-left corner.

https://i.imgur.com/a6hfmnR.png

**Comment by kprinssu** [comment]
Have you tried click and dragging the circle?

I am closing this issue because it is not reproducible.

**Comment by ddejohn** [comment]
Tried clicking and dragging, colors change but the cursor is still stuck. 1.0.5, and also in 1.0.6.

**Comment by ethanmsmith** [comment]
I did come across this, but it only happened on my dev computer (vms and other computers had no problem) and just assumed it was my computer acting strange. I don't think its a problem with the control since this is an official microsoft xaml control. 

Any additional info like what you did before trying to change the color would be helpful.

**Comment by ddejohn** [comment]
Installed the app, went to the color-changing tab, tried to change the colors. Wasn't working in 1.0.5, and also not working in the latest release.

**Comment by kprinssu** [comment]
Going to reopen the issue. I’ll need to take a look at why the issue occurs. 

@ddejohn can you provide more info on the issue? Did it work before, what’s your OS? Does older versions of the app work?

**Comment by ddejohn** [comment]
I just got the keyboard a few days ago. 1.0.5 was my first version of the app, and it never worked. Running Windows 10 Education (basically Enterprise).

**Comment by kprinssu** [comment]
So the app works without issues (see https://github.com/kprinssu/anne-keyboard-windows/issues/39#issuecomment-393337837)?

**Comment by ethanmsmith** [comment]
That's interesting.. My dev computer is win10 education, but other machines I tested on are not

**Comment by ddejohn** [comment]
@kprinssu no, the RGB cursor is still stuck. Everything else is working as far as I can tell. Are there logs being dumped somewhere I can provide?

https://i.imgur.com/XTJLnsz.gifv

**Comment by ddejohn** [comment]
@kprinssu no, the RGB cursor is still stuck. Everything else is working as far as I can tell. I posted a GIF.

**Comment by kprinssu** [comment]
The logs are not being saved to file. It should be streamed to standard out. As far as I know, on release builds of the apps it’s not captured by anything.

I’ll grab a copy of Win10 Education and debug further on my own.
